model/eligibility/Eligibility.php

```diff
@@ -137,6 +138,15 @@ public function getId()
         return $this->id;
     }
 
+    /**
+     * @return bool
+     * @throws \core_kernel_persistence_Exception
+     */
+    public function getProctored(){
```

Comment:
Please put the `{` in a new line.

Reply:
Also if it returns a boolean, the function name must reflect it, so what do you think about `isProctored()`?